packages/webapp/src/views/ConsolidatedTreemap/index.jsx

```diff
@@ -27,7 +27,6 @@ const Markup = ({ items, initialSelected }) => (
       disabled: "2019-20",
     }}
     anchor="consolidated-treemap"
-    isConsolidatedChart
```

Comment:
this isn't cool - you're removing the argument saying that this is the consolidated chart, from a component called ConsolidatedTreemap...

Reply:
Could you suggest a more appropriate solution? Is this to allow the button to show?

Reply:
I've removed the logic that takes this boolean now, so that it doesn't look redundant